commitizen/defaults.py

```diff
@@ -57,6 +57,9 @@ class Settings(TypedDict, total=False):
     prerelease_offset: int
     encoding: str
     always_signoff: bool
+    format: str | None
```

Comment:
I think it might be better for us to change it to `changelog_format` as well

Reply:
Renamed into `changelog_format` 👍🏼